TCMSApp/src/client/app/defects/defects.controller.spec.js

```diff
@@ -80,6 +83,22 @@ describe('Tests Defects', function() {
             expect(controller.currentDefect.description).to.equal('Description14122444dghv4c2');
         });
     });
+    beforeEach(function() {
```

Comment:
empty before each block? why?
One note. can you please check other beforeEach blocks in this spec file. There are a lot of them on the same level. I guess they are all executed before every test. So why shouldn't we merge them in one somehow.